### ISSUE TITLE
Feature Request: Scoll bar arrows in RStudio themes "Modern" and "Sky"

### ISSUE THREAD

**Issue by nwo9** [rstudio/rstudio#4594]
****Feature Request:**** I'd wish to have clickable scroll bar arrows for source and console in RStudio when using the "modern" or the "Sky" theme.

****Problem:**** I prefer a dark editor theme with either the "Modern" or the "Sky" RStudio theme. Both RStudio themes, however, remove the clickable arrows from the scroll bar for the console and source pane. Without these, it is very difficult to navigate to a specific line upwards, especially in the console. (See "How to reproduce the problem" below.)
The "classic" option for the RStudio theme has clickable scroll bar arrows but  leaves most of the panes and navigation bars in bright colors. (See screen shots below). 

![screenshot-classic](https://user-images.githubusercontent.com/49400844/55726398-ff8dc080-5a0f-11e9-91e4-d790b1f7cba0.png)
![screenshot-modern](https://user-images.githubusercontent.com/49400844/55726393-fdc3fd00-5a0f-11e9-8caa-4a5d232d62fe.png)

****How to reproduce the problem:**** 
In RStudio (my version:  1.1.463) for Windows, go to 

Tools --> Global Options --> Appearance.

Select a dark editor theme (e.g. Monokai) and  the RStudio theme "Modern".
If necessary (e.g. if you were on "classic" before) restart RStudio.
Enter in the console:
`for(x in 1:10000) {print(x)}`
After execution, try to scroll upwards such that a specific number, e.g. 9500 shows up on the top line within your console pane. It's gonna be quite tricky:  The mouse is getting very wiggly with this amount of output and the up and down arrows won't help you because they recall the code you just entered.

**Comment by ronblum** [comment]
@nwo9 RStudio Desktop 1.2 is now available at https://www.rstudio.com/products/rstudio/download/ . Does the problem still occur for you in the current release?

**Comment by nwo9** [comment]
Thanks for coming back on this! Quite a coincidence that I put this up the day you had a new release.

Concerning __repro: Yes,__  the problem still exists. See the screenshot from version 1.2.1335 attached.

![screenshot-modern-2](https://user-images.githubusercontent.com/49400844/55867390-adb97780-5b82-11e9-9a49-13df6249e6cf.png)

It is better than before because the slider is reducing further then just to a ball, but it is still difficult to move the window to an exact position when there has been a lot of output in the console (equivalently, in very long source files, e.g. such as RMarkdown files for long articles). In my example (similar sized source and console pane, 10k lines of output in the console, textsize =10pt) I could move up and down in steps of five lines with very careful movements of the mouse. In long coding sessions, recalling earlier output thus gets difficult.

The filter in the dataviewer is a great improvement! I am wondering however why the slider is a small dot instead of a vertical line as for the console. Anyway, this is a minor and separate issue.

Thanks a ton for the great work!

**Comment by nwo9** [comment]
(sorry for the unintended closing and reopening)

**Comment by nwo9** [comment]
The issue still has the "repro needed" label. Is this accurate? I think the repro should be there. Could I remove the label?

**Comment by gtritchie** [comment]
Definitely easy to repro with both 1.2 and post 1.2.

**Comment by nwo9** [comment]
Just a ping to bring this up again. Stale bot has marked it due to lack of activity, but the problem continues (RStudio 1.2.1335, Linux system). In fact, it had been brought up again [here](https://github.com/rstudio/rstudio/issues/6477) but there got closed as a duplicate. So I clearly don't think this is stale. It's a small but important flaw which should be easy to address, I hope. (at least the classic theme had arrows).

**Comment by ronblum** [comment]
This is still reproducible in 2021.09.2+382 on Windows 10.

@gtritchie I marked this as Electron for the moment, but do you think Electron could help with this?

**Comment by gtritchie** [comment]
No, this behavior isn't changed by the switch to Electron. I confirmed with a local build on Windows.

**Comment by shivam7898** [comment]
Any Progress on this one?
All the editor themes which have dark background show this problem of no up/down arrows and thin scroll bar which is difficult to click

**Comment by nwo9** [comment]
I would like to support this again -  the bug is still there after all these updates. I know people who avoid dark themes due to this.
